shared/src/Twin.cs

```diff
@@ -151,6 +151,19 @@ public Twin(TwinProperties twinProperties)
         [JsonProperty(DefaultValueHandling = DefaultValueHandling.IgnoreAndPopulate)]
         public X509Thumbprint X509Thumbprint { get; internal set; }
 
+        /// <summary>
```

Comment:
Add same details xml comments here as well? Is this even settable or should one set this value in Twin at all?

Reply:
As far as I can tell, you can set it and it does nothing. So I made these two properties readonly for customers. I decided not to copy over all the doc comments from Device.cs because most of it is about how you set it. Honestly, I don't understand why it is duplicated in the Twin, and I may end up removing it based on feedback from Anca.